src/components/dropdown-menu/home-dropdown-menu.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Note } from '../../data-management/types';
+import { DropdownMenu, DropdownMenuItem, DropdownMenuSeparator } from './dropdown-menu';
+import styles from './dropdown-menu.module.scss';
+import { useNotesStore } from '../../data-management/use-notes-store';
+
+export const HomeDropdownMenu = ({ id, isPinned, title, description }: Omit<Note, 'createdAt' | 'modifiedAt'>) => {
+    const { addNote, updateNote, deleteNote, syncNotes, pinNote, unPinNote } = useNotesStore();
+
+    const handlePinNote = () => {
+        pinNote(id);
+    };
+
+    const handleUnpinNote = () => {
+        unPinNote(id);
+    };
+
+    const handleDuplicateNote = () => {
+        addNote({ title, description });
+    };
+
+    const handleDelete = () => {
+        deleteNote(id);
+    };
+
+    const handleNewNote = () => {
```

Comment:
I am not sure what does it mean.

If it's a menu for an existing note, then i'm not sure there should be 'New note' menu item

Reply:
Sounds reasonable, @RanBerkovich1 wdyt?

Reply:
We can remove it